Running dev fails

Running  `docker-compose -f docker-compose.yml -f docker-compose.dev.yml up --build` as mentioned in the docs fails with:

```
frontend_1  |
frontend_1  | > checkers-frontend@0.1.0 start /checkers/frontend
frontend_1  | > webpack-dev-server
frontend_1  |
frontend_1  | sh: webpack-dev-server: not found
frontend_1  | npm ERR! code ELIFECYCLE
frontend_1  | npm ERR! syscall spawn
frontend_1  | npm ERR! file sh
frontend_1  | npm ERR! errno ENOENT
frontend_1  | npm ERR! checkers-frontend@0.1.0 start: `webpack-dev-server`
frontend_1  | npm ERR! spawn ENOENT
frontend_1  | npm ERR!
frontend_1  | npm ERR! Failed at the checkers-frontend@0.1.0 start script.
frontend_1  | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
frontend_1  | npm WARN Local package.json exists, but node_modules missing, did you mean to install?
```

--- a/README.md
+++ b/README.md
@@ -11,7 +11,8 @@ Run the app with
 docker-compose -f docker-compose.yml -f docker-compose.prod.yml up --build
 ```
 
-or in development mode (hot reloading when source code changes) with
+or in development mode (mounts the source code into the container so that
+changes are reflected) with
 
 ```sh
 docker-compose -f docker-compose.yml -f docker-compose.dev.yml up --build

--- a/docker-compose.dev.yml
+++ b/docker-compose.dev.yml
@@ -1,4 +1,4 @@
-version: "3.7"
+version: '3.7'
 
 services:
   redis:
@@ -9,9 +9,7 @@ services:
     ports:
       - 8000:8000
     volumes:
-      - type: bind
-        source: ./backend
-        target: /checkers/backend
+      - ./backend:/checkers/backend
   frontend:
     build:
       target: build
@@ -21,6 +19,5 @@ services:
     ports:
       - 80:9000
     volumes:
-      - type: bind
-        source: ./frontend
-        target: /checkers/frontend
+      - ./frontend/public:/checkers/frontend/public
+      - ./frontend/src:/checkers/frontend/src

--- a/frontend/Dockerfile
+++ b/frontend/Dockerfile
@@ -2,10 +2,10 @@
 FROM node:14-alpine as build
 ARG CHECKERS_BACKEND="localhost:8000"
 WORKDIR /checkers/frontend
-ENV PATH /checkers/frontend/node_modules/.bin:$PATH
 COPY package* ./
 RUN npm ci --silent
 COPY . ./
+ENV PATH /checkers/frontend/node_modules/.bin:$PATH
 RUN CHECKERS_BACKEND=${CHECKERS_BACKEND} npm run build
 
 FROM nginx:alpine

